### DISCUSSION TITLE
Why coveralls when you already have Jest?

### DISCUSSION THREAD

**Discussion by fernandocanizo** [hagopj13/node-express-boilerplate#99] [Q&A] [answered]
Hi all, firstly: great work @hagopj13 :)

Question: why you decided to go with [Coveralls](https://coveralls.io/) when you already can use Jest coverage? What are the benefits?

In my opinion paid solutions should be left as optionals, or maybe another repo. You could have the totally free one (as in free beer and free speech too) and a more complete one with tools like this one.

**Comment by hagopj13** [comment, marked as answer]
@fernandocanizo I am indeed using Jest to create `text-lcov` files which are then sent to coveralls.

Coveralls is being used only to display the current coverage percentage. It can be easily taken out.
